.github/workflows/code_freeze_start.yml

```
@@ -0,0 +1,67 @@
+name: Start code freeze
+
+on:
+  workflow_dispatch:
+  milestone:
+    types: [opened, created, edited]
+
+jobs:
+  start_code_freeze:
+    if: | 
+      github.event_name == 'workflow_dispatch' || 
+      (github.event.milestone.title == 'Core Freeze' && github.event.milestone.state == 'open') 
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+    steps:
+    - uses: octokit/request-action@v2.x
+      name: 'Create Code Freeze Milestone'
+      id: milestones
+      if: github.event_name == 'workflow_dispatch'
+      with:
+        route: POST /repos/{owner}/{repo}/milestones
+        owner: Datadog
```

not sure if it is case sensitive

```suggestion
        owner: DataDog
```